### ISSUE TITLE
get author from sitemetadata

### ISSUE THREAD

**Issue by swyxio** [syntra/gatsby-remark-social-cards#19]
i have author in my sitemetadata, not in my frontmatter.. would you be open to pulling data for that?

**Comment by ariamckinley** [comment]
Yeah I'd definitely be open to that :) Sorry for the delayed response, I haven't worked on this plugin in a bit. I could probably add that in sometime this week. If you'd like to open a PR yourself, I'd definitely merge it. Just LMK, if I don't hear anything I'll work on it next chance I get

**Comment by ariamckinley** [comment]
I was thinking about this a bit more since commenting, and I think I have an idea of how this would best work. The author should be pulled from sitemetadata *first* and then if an author is passed in the frontmatter it will override the author in sitemetadata for that single post.

Let me know if there's any issues with approaching it this way.

**Comment by swyxio** [comment]
not really. i also moved away from this :)
